src/js/base/module/Editor.js

```diff
@@ -452,9 +452,13 @@ export default class Editor {
       if (this.context.invoke(eventName) !== false) {
         event.preventDefault();
         // if keyMap action was invoked
-        return true;
+        context.invoke(eventName);
+//        return true;
```

Comment:
I'm not sure this is an intended commenting out.

Reply:
It was, in case it needed to be changed. I didn't see any lint errors locally.